src/services/ContributingDeveloperAuditService/scm/GitHub/GitHubService.ts

```diff
@@ -0,0 +1,30 @@
+import GitHubApiClient, {
+  ContributingDeveloper,
+  GitHubOrganization,
+  GitHubRepository,
+} from "./api/GitHubApiClient";
+
+class GitHubService {
```

Comment:
if there is no logic in just have the audit provider call the api client directly